Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/ParametersSectionTranslator.swift

```diff
@@ -20,15 +20,27 @@ struct ParametersSectionTranslator: RenderSectionTranslator {
         translateSectionToVariantCollection(
             documentationDataVariants: symbol.parametersSectionVariants
         ) { _, parameters in
-            guard !parameters.parameters.isEmpty else { return nil }
+            guard !(parameters.parameters.isEmpty || parameters.parameters.count < 1) else {
```

Comment:
Why do you check for `isEmpty` or `count < 1` ?

```suggestion
            guard !parameters.parameters.isEmpty else {
```

Reply:
Good catch, I think this was old code that I had left when I was checking whether `isEmpty` was working or not